sublime_linter.py

```diff
@@ -44,6 +44,12 @@ def plugin_loaded():
     log_handler.install()
     backup_old_settings()
 
+    from package_control import events
+    if events.install('SublimeLinter'):
+        util.message('SublimeLinter has been installed. Please restart Sublime Text.')
```

Comment:
Why would you need to restart when SL is installed? I mean, unless you already had other linter plugins installed that depend on it.

Reply:
I guess only if you already had some linter plugins installed and then (re-) installed SL.